app/src/main-process/squirrel-updater.ts

```diff
@@ -18,6 +18,9 @@ const exeName = Path.basename(process.execPath)
  */
 export function handleSquirrelEvent(eventName: string): Promise<void> | null {
   switch (eventName) {
+    case '--squirrel-firstrun':
+      return Promise.resolve()
```

Comment:
Right, this would require adding a bit more nuance than `handleSquirrelEvent` currently supports.

The log message doesn't bother me. It's up to you whether it's worth doing the refactor 😁